tests/src/test/scala/runtime/actionContainers/PythonActionContainerTests.scala

```diff
@@ -267,10 +267,7 @@ class PythonActionContainerTests extends BasicActionRunnerTests with WskActorSys
       val (out, err) = withActionContainer() { c =>
         val (initCode, initRes) = c.init(initPayload(code, main = "main"))
         if (initErrorsAreLogged) {
-          initCode should be(200)
-          val args = JsObject("msg" -> JsString("any"))
-          val (runCode, runRes) = c.run(runPayload(args))
-          runCode should be(502)
+          initCode should be(502)
```

Comment:
init code of 502 means an error which will cause the invoke to destroy the container. I do not understand the change to this test.

Reply:
Initialization phase will execute part of the code. In this test scenario, initcode should be 502.
thx.

Reply:
> init code of 502 means an error which will cause the invoke to destroy the container. I do not understand the change to this test.

Initialization phase will execute part of the code. In this test scenario, initcode should be 502.

Reply:
Thanks @silvasong for the clarification.